File: src/main/java/org/aldo/api/data/dto/DailyDetailDto.java
package org.aldo.api.data.dto;

import lombok.Data;

import java.util.List;

@Data
public class DailyDetailDto {
    private SummaryProjectDto project;
    private MonthlyHoursDto monthlyHours;
    private List<DailyHoursDto> dailyHours;
}

File: src/app/model/dto/DailyDetailDto.ts
import {DailyHoursDto} from "./DailyHoursDto";
import {SummaryProjectDto} from "./SummaryProjectDto";
import {MonthlyHoursDto} from "./MonthlyHoursDto";

export class DailyDetailDto {
  project: SummaryProjectDto;
  monthlyHours: MonthlyHoursDto;
  dailyHours: DailyHoursDto[];
  constructor(data: any) {
    this.project = data.project;
    this.monthlyHours = data.monthlyHours;
    this.dailyHours = data.dailyHours
  }
}
